app/data/models/payment-request.js

```diff
@@ -44,6 +44,11 @@ module.exports = (sequelize, DataTypes) => {
       foreignKey: 'paymentRequestId',
       as: 'settlements'
     })
+    paymentRequest.hasMany(models.settlement, {
+      sourceKey: 'invoiceNumber',
+      foreignKey: 'invoiceNumber',
+      as: 'settlementsByInvoiceNumber'
+    })
```

Comment:
Cannot do join in Sequelize without association.  This is not actually creating a relationship in the database, more just allowing Sequelize to understand how to do a join on `invoiceNumber`.